server/routes.js

```diff
@@ -416,22 +401,32 @@
         var project_name = req.body.params.project_name;
         var file_name = req.body.params.file_name;
         var code = req.body.params.file_code;
-        if (!project_name || !file_name || !code) {
-            res.status(400).send("Failed to get project, file name or code");
+        if (!project_name || !file_name) {
+            res.status(400).send("Failed to get project, file name");
```

Comment:
I didn't get how this would impact in anything, but in the same phrase I think I should replace , to or.

Since, it is two statements now